emitter/gkelog/severity.go

```diff
@@ -0,0 +1,57 @@
+package gkelog
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/vimeo/alog/v3"
+)
+
+// Severity levels
+const (
+	SeverityDefault   = "DEFAULT"   // The log entry has no assigned severity level.
+	SeverityDebug     = "DEBUG"     // Debug or trace information.
+	SeverityInfo      = "INFO"      // Routine information, such as ongoing status or performance.
+	SeverityNotice    = "NOTICE"    // Normal but significant events, such as start up, shut down, or a configuration change.
+	SeverityWarning   = "WARNING"   // Warning events might cause problems.
+	SeverityError     = "ERROR"     // Error events are likely to cause problems.
+	SeverityCritical  = "CRITICAL"  // Critical events cause more severe problems or outages.
+	SeverityAlert     = "ALERT"     // A person must take an action immediately.
+	SeverityEmergency = "EMERGENCY" // One or more systems are unusable.
+)
+
+// Separate private function so that LogSeverity will have the correct calldepth
```

Comment:
This is a little hard to understand (might be useful to say why calldepth 3 is correct), and also it isn't only called by `LogSeverity` but all the other functions here.

Reply:
Updated the comments